.github/labeler.yml

```diff
@@ -1,13 +1,14 @@
 kind/docs:
   - "**/*.md"
```

Comment:
`- "**.md"`

Reply:
Should be same outcome, feel free to make another PR